docs/runtimes/private-repo.ipynb

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
<p>The .yaml file will be stored in a path like this: /function-name/function.yaml (e.g /func/function.yaml) - maybe add how it look from the repo point of view, for example: /repo/function-name/function.yaml or https://raw.githubusercontent.com/user name/repo name/function-name/function.yaml"</p><p> </p><p>another point that can be added - in the function folder the one that old the function.yaml file you can stored any additional file for example: source file, notebook example </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>